doc/references.bib

```diff
@@ -51,8 +51,7 @@ @article{BailletEtAl2001
   pages = {14-30},
   title = {Electromagnetic Brain Mapping},
   volume = {18},
-  year = {2001},
-  doi = {10.1109/MMSP.2010.5662067}
```

Comment:
Oops, I meant to add that DOI to BarachantEtAl2010...

Reply:
Done